homeassistant/components/media_player/bluesound.py

```diff
@@ -669,15 +781,96 @@ def supported_features(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
+        sleep = 0
+        if self._status is not None:
+            sleep = self._status.get('sleep', 0)
+
         return {
-            ATTR_MODEL: self._model,
-            ATTR_MODEL_NAME: self._model_name,
-            ATTR_BRAND: self._brand,
+            ATTR_SLEEP: sleep
```

Comment:
Let's remove it. We should not have automatic decreasing values in state attributes. Ok values would be a sleep started timestamp and total sleep seconds.

Reply:
Ok, I'll remove this